tests/features/community_content/add_community_content.feature

```diff
@@ -46,7 +46,6 @@ Feature: Add community content
     And I click "Add <content type>"
     Then I should see the heading "Add <content type>"
     But I should not see the following lines of text:
-      | Authored by                  |
```

Comment:
This doesn't provide enough test coverage. I removed the access to the field and the tests were still green.
Also we should extend the tests to cover the video and newsletter content types.